microprofile/opentracing-extension/src/main/java/org/wildfly/extension/microprofile/opentracing/TracingDeploymentProcessor.java

```diff
@@ -110,10 +168,30 @@ private JBossWebMetaData getJBossWebMetaData(DeploymentUnit deploymentUnit) {
             // not a web deployment, nothing to do here...
             return null;
         }
-
         return warMetaData.getMergedJBossWebMetaData();
     }
 
+    private String getTracerConfiguration(DeploymentPhaseContext deploymentPhaseContext) {
+        DeploymentUnit deploymentUnit = deploymentPhaseContext.getDeploymentUnit();
+        JBossWebMetaData jbossWebMetaData = getJBossWebMetaData(deploymentUnit);
+        if (null == jbossWebMetaData || null == jbossWebMetaData.getContextParams()) {
+            return null;
+        }
+        for (ParamValueMetaData param : jbossWebMetaData.getContextParams()) {
+            if (TracerInitializer.SMALLRYE_OPENTRACING_TRACER_CONFIGURATION.equals(param.getParamName())) {
```

Comment:
Having not read the opentracing spec thoroughly...
Is this really how users should define opentracing configuration?  I would expect this configuration to be defined via MP Config. Configuration via servlet context parameters is very retro.

Reply:
This is how this was done in the initial integration. Tracer configuration is not really specified in opentracing specs.

Reply:
To align with vendor-specific configuration for other MP specs, I would suggest using vendor specific MP config namespace. e.g. "mp.opentracing.extensions.tracer.configuration" (or similar).

Reply:
Except for those that were already defined